app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  after_action :verify_authorized
+  def index
+    authorize :user
```

Comment:
```suggestion
    authorize User
```
Let's just use class of objects we are indexing

Reply:
@rafalrothenberger done